src/overlays/actors/ovl_En_Mkk/z_en_mkk.c

```diff
@@ -36,18 +47,36 @@ const ActorInit En_Mkk_InitVars = {
     (ActorFunc)EnMkk_Draw,
 };
 
-// static ColliderSphereInit sSphereInit = {
-static ColliderSphereInit D_80A4F720 = {
-    { COLTYPE_HIT3, AT_NONE | AT_TYPE_ENEMY, AC_NONE | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_SPHERE, },
-    { ELEMTYPE_UNK0, { 0xF7CFFFFF, 0x00, 0x04 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_HARD, BUMP_ON, OCELEM_ON, },
+static ColliderSphereInit sSphereInit = {
+    {
+        COLTYPE_HIT3,
+        AT_NONE | AT_TYPE_ENEMY,
+        AC_NONE | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_SPHERE,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0xF7CFFFFF, 0x00, 0x04 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_HARD,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 1, { { 0, 0, 0 }, 15 }, 100 },
 };
 
-// sColChkInfoInit
-static CollisionCheckInfoInit D_80A4F74C = { 1, 15, 30, 10 };
+struct EnMkkDlists {
+    /* 0x00 */ Gfx* unk0;
+    /* 0x04 */ Gfx* unk4;
+    /* 0x08 */ Gfx* unk8;
+    /* 0x0C */ Gfx* unkC;
+}; // size = 0x10
```

Comment:
```suggestion
typedef struct EnMkkDLists {
    /* 0x0 */ Gfx* unk0;
    /* 0x4 */ Gfx* unk4;
    /* 0x8 */ Gfx* unk8;
    /* 0xC */ Gfx* unkC;
}; EnMkkDLists// size = 0x10
```
With this change you will no longer need to use `struct EnMkkDLists` so please remove them.

Reply:
Quick question as well, does this work as just an array of `Gfx*` instead of a new struct?

Reply:
no, sorry it doesn't work. Messes up the asm in the last Draw functions